### ISSUE TITLE
Is this repo still being mantained

### ISSUE THREAD

**Issue by dturton** [Code2Life/http-adapter#31]
Looks super interesting. Are you still working on this?

**Comment by Code2Life** [comment]
actually not, but I have some more ideas based on this, and I did see a much greater project / company that has similar thoughts : https://github.com/fermyon/spin

**Comment by dturton** [comment]
I didn't. Is it generally the same idea? My use case is to integrate services via web hooks.

**Comment by Code2Life** [comment]
not really, but spin has much larger scope than http-adapter. the functionality of this repo is fine, but dependencies are not upgraded since 2 or 3 years ago, plz care of vulnerabilities of this repo.
